packages/graphql/tests/integration/issues/label-cypher-injection.test.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { GraphQLSchema } from "graphql";
+import { graphql } from "graphql";
+import { Neo4jGraphQLAuthJWTPlugin } from "@neo4j/graphql-plugin-auth";
+import type { Driver, Session } from "neo4j-driver";
+import Neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src";
+import { generateUniqueType } from "../../utils/graphql-types";
+import { createJwtRequest } from "../../utils/create-jwt-request";
+
+describe("Label cypher injection", () => {
+    let schema: GraphQLSchema;
+    let driver: Driver;
+    let neo4j: Neo4j;
+    let session: Session;
+
+    beforeAll(async () => {
+        neo4j = new Neo4j();
+        driver = await neo4j.getDriver();
+    });
+
+    beforeEach(async () => {
+        session = await neo4j.getSession({ defaultAccessMode: "WRITE" });
```

Comment:
question (non-blocking): Is the `session` really needed for this particular test case? (I've never used a label from the context or the JWT token)

Reply:
Well, this is just to pass the graphql test with the correct bookmark. I think it is the best way to avoid race conditions.